qkc-fork.yaml

```diff
@@ -123,19 +125,42 @@ def:
             - "op-e2e/bindings/soulgastoken.go"
             - "op-e2e/sgt/sgt_test.go"
             - "op-e2e/sgt/helper.go"
+            - "op-e2e/actions/sgt/sgt_test.go"
+            - "op-e2e/config/init.go"
         - title: "L2 Blob e2e test"
           description: ""
           globs:
             - "op-e2e/l2blob/l2blob_test.go"
+        - title: "Script to run overall local test"
+          description: |
+            This script is used to run the overall local test for the OP-stack.
+            For details, please refer to [here](https://github.com/ethstorage/optimism/pull/161).
+          globs:
+            - ".mise-tasks/dev-test.sh"
+    - title: "Tools"
+      sub:
+        - title: "Script to upgrade SoulGasToken"
+          description: |
+            This script is used to upgrade the SoulGasToken contract on L2.
+            It allows for the deployment of a new version of the contract, ensuring that
+            users can benefit from the latest features and improvements.
+          globs:
+            - "packages/contracts-bedrock/scripts/deploy/UpgradeSoulGasToken.s.sol"
+        - title: "Script to update the AnchorStateRegistry"
+          description: |
+            This script is used to update the AnchorStateRegistry with a new anchor state when
+            switching from permissioned FDG to permissionless FDG on L1.
+            Fore details, please refer to [here](https://github.com/ethstorage/pm/blob/main/L2/beta_testnet_fdg_upgrade.md).
+          globs:
+            - "packages/contracts-bedrock/scripts/deploy/UpgradeAnchorStateRegistry.s.sol"
+
 
 # ignored globally, does not count towards line count
 ignore:
   - ".circleci/*"
   - "*.sum"
   - "go.mod"
-  - "fork.yaml"
-  - "Makefile"
-  - ".golangci.yml"
+  - "es-fork.yaml"
```

Comment:
change to qkc-fork.yaml?

Reply:
Done in e6d89155d8b7df3d7e4338129c2977c0bbde8956.